Library/SPIClient.m

```diff
@@ -1991,6 +2005,7 @@ - (void)onReadyToTransact {
             dispatch_async(self.queue, ^{
                 if (!self.hasSetPosInfo) {
                     [weakSelf callSetPosInfo];
+                    weakSelf.transactionReport = [SPITransactionReportHelper createTransactionReportEnvelope:weakSelf.posVendorId posVersion:weakSelf.posVersion libraryLanguage:@"ios" libraryVersion:[SPIClient getVersion] serialNumber:weakSelf.serialNumber];
```

Comment:
Sorry the JS commit is a bit messy here, but can you please update the references to `libraryLangauge` to be a property of the `SPIClient` and also update the parameter in the callSetPosInfo too?